python/paddle/v2/dataset/tests/vocseg_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.v2.dataset.voc_seg
```

Comment:
The PEP8 style https://www.python.org/dev/peps/pep-0008/#package-and-module-names says:

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. 

Here the underscore doesn't really improve the readability, how about naming it `voc2012` instead?